bin/plotting/pycbc_faithsim_plots

```diff
@@ -0,0 +1,221 @@
+
+    return {
+        "total_mass": M,
+        "mass_ratio": q,
+        "mchirp": mchirp,
+        "spin1_magnitude": s1,
+        "spin2_magnitude": s2,
+        "eta": eta,
+        "chi_eff": chieff,
+        "s81": s81,
+        "s82": s82,
+    }
+
+
+def basic_scatter(
+    out_name,
+    neg_idx,
+    xname,
+    yname,
+    title,
+    xval,
+    yval,
+    cval,
+    cname,
+    vmin,
+    vmax,
+    xmin,
+    ymin,
+    majorL,
+    minorL,
+    colormap,
+):
+    if colormap != "jet":
+        cmap = colormap
+    else:
+        cmap = matplotlib.cm.jet
+        cmap.set_under(color="gray")
+
+    if vmin is not None:
+        vmin = float(vmin)
+
+    if vmax is not None:
+        vmax = float(vmax)
+
+    fig = pylab.figure(num=None)
+    pylab.scatter(
+        xval,
+        yval,
+        c=cval,
+        linewidths=0,
+        s=3,
+        vmin=vmin,
+        vmax=vmax,
+        cmap=cmap,
+        alpha=0.7,
+    )
+
+    if len(neg_idx) > 0:
+        pylab.scatter(
+            xval[neg_idx],
+            yval[neg_idx],
+            c="red",
+            marker="x",
+            label="Unable to generate waveforms",
+        )
+        matplotlib.pyplot.legend()
+
+    if cval is not None:
+        bar = pylab.colorbar()
+        bar.set_label(cname)
+
+    pylab.xlabel(xname)
+    pylab.ylabel(yname)
+
+    if xmin is None:
+        xmin = min(xval)
+    else:
+        xmin = float(xmin)
+
+    if ymin is None:
+        ymin = min(yval)
+    else:
+        ymin = float(ymin)
+
+    pylab.xlim(xmin, max(xval))
+    pylab.ylim(ymin, max(yval))
+
+    ax = fig.gca()
+    if majorL:
+        majorL = float(majorL)
+        ax.xaxis.set_major_locator(MultipleLocator(majorL))
+        ax.yaxis.set_major_locator(MultipleLocator(majorL))
+    if minorL:
+        minorL = float(minorL)
+        ax.xaxis.set_minor_locator(MultipleLocator(minorL))
+        ax.yaxis.set_minor_locator(MultipleLocator(minorL))
+
+    pylab.grid()
+    pylab.title(title)
+
+    pylab.savefig(out_name, dpi=500)
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument(
+    "--input-file",
+    required=True,
+    help="resulting .dat file from pycbc_faithsim_collect_full_results",
+)
+parser.add_argument("--parameter-x", required=True)
+parser.add_argument("--parameter-y", required=True)
+parser.add_argument("--parameter-z", required=True)
+parser.add_argument("--colormap", default="jet")
+parser.add_argument("--vmin", default=None, help="min value of the colorbar")
+parser.add_argument("--vmax", default=None, help="max value of the colorbar")
+parser.add_argument("--xmin", default=None)
+parser.add_argument("--ymin", default=None)
+parser.add_argument("--majorL", default=None)
+parser.add_argument("--minorL", default=None)
+parser.add_argument("--output-plot", required=True, help="name of the output plot")
+
+args = parser.parse_args()
+
+with open(args.input_file, "r") as f:
+    l = f.readlines()[0]
```

Comment:
This is not necessary. After you use `genfromtxt()` below, `data.dtype.names` gives you the list of field names.